### DISCUSSION TITLE
Does Elyra require Jupyter Enterprise Gateway to run on Jupyterhub?

### DISCUSSION THREAD

**Discussion by duongnt** [elyra-ai/elyra#2080] [Q&A] [answered]
From https://elyra.readthedocs.io/en/latest/recipes/deploying-elyra-in-a-jupyterhub-environment.html, looks like it's recommended to use Jupyter Enterprise Gateway, but it's not clear to me why? We're not using it, is it still possible to run elyra with our jupyterhub deployment?

I just learned about Elyra a couple days ago, so apologies for the basic questions.

**Comment by lresende** [comment, marked as answer]
No, Elyra does not require Enterprise Gateway to run (standalone and/or hosted by JupyterHub.

When using Enterprise Gateway, your kernels have the ability to be launched as a separate pod and used only throughout the kernel duration, hence providing you a better resource allocation. Otherwise, all the resources need to be allocated at the launch of Elyra and will be in use for the duration of your environment.

**Reply by duongnt** [reply]
Thanks @lresende.
